go-controller/pkg/util/net_linux_unit_test.go

```diff
@@ -657,6 +679,17 @@ func TestLinkRouteExists(t *testing.T) {
 				call.Once()
 			}
 
+			for _, item := range tc.onRetArgsLinkIfaceOpers {
+				call := mockLink.On(item.onCallMethodName)
```

Comment:
Please add `mockLink.AssertExpectations(t)` below the line(L703) `mockNetLinkOps.AssertExpectations(t)`

Reply:
@girishmg @cathy-zhou looks good to me if the above comment is addressed

Reply:
@vjayaramrh thanks for your comments. I've fixed it and repushed.